src/app/components/student/student.component.ts

```diff
@@ -26,6 +26,19 @@ export class StudentComponent implements OnInit {
   isLoading = false;
   currentFilterArray: Array<FilterModel> = [];
   headers = studentsTableHeaders.data;
+  rowData: Student[] = [];
+
+  columnDefs = [
```

Comment:
can you set Headers in the same way as a Students?
create const for table Headers to use it in ag-grid.